frontend/src/stores/feature-flag.ts

```diff
@@ -131,8 +131,23 @@ const FEATURE_FLAG = "feature_flag"
 
 export const useFeatureFlagStore = defineStore(FEATURE_FLAG, {
   state: () => {
-    const deploymentEnv = useRuntimeConfig().public.deploymentEnv as DeployEnv
-    return initializeFlagState(deploymentEnv)
+    const nuxtApp = tryUseNuxtApp()
```

Comment:
`useRuntimeConfig` throws an error when the Nuxt context isn't available. The same is true for `useNuxtApp`. This is why we use `tryUseNuxtApp` here, which returns `null` instead of throwing an error.

Reply:
Could we reformat this a bit?

```js
  const nuxtApp = tryUseNuxtApp();
  
  if (!nuxtApp) {
    throw new Error(
      "Could not set up feature flag store because Nuxt content isn't available."
    );
  }
  
  return initializeFlagState(
    nuxtApp.$config.public.deploymentEnv as DeployEnv
  );
```